README.md

```diff
@@ -292,169 +292,7 @@ JWKS is supported via [get-jwks](https://github.com/nearform/get-jwks). Check ou
-╚═════════════════════════════════════╧═════════╧══════════════════╧═══════════╧═════════════════════════╝
-
-╔═════════════════════════════════════╤═════════╤══════════════════╤═══════════╤═════════════════════════╗
-║ Slower tests                        │ Samples │           Result │ Tolerance │ Difference with slowest ║
-╟─────────────────────────────────────┼─────────┼──────────────────┼───────────┼─────────────────────────╢
-║ PS512 - jsonwebtoken (async)        │    2500 │   7240.21 op/sec │  ± 0.89 % │                         ║
-║ PS512 - jsonwebtoken (sync)         │    2000 │   7449.38 op/sec │  ± 0.91 % │ + 2.89 %                ║
-║ PS512 - fast-jwt (async)            │    1500 │   8301.99 op/sec │  ± 0.81 % │ + 14.67 %               ║
-║ PS512 - jose (sync)                 │    4000 │  11944.57 op/sec │  ± 0.99 % │ + 64.98 %               ║
-║ PS512 - fast-jwt (sync)             │    1000 │  12881.96 op/sec │  ± 0.76 % │ + 77.92 %               ║
-║ PS512 - fast-jwt (async with cache) │   10000 │ 155603.59 op/sec │  ± 4.27 % │ + 2049.16 %             ║
-╟─────────────────────────────────────┼─────────┼──────────────────┼───────────┼─────────────────────────╢
-║ Fastest test                        │ Samples │           Result │ Tolerance │ Difference with slowest ║
-╟─────────────────────────────────────┼─────────┼──────────────────┼───────────┼─────────────────────────╢
-║ PS512 - fast-jwt (sync with cache)  │   10000 │ 172097.91 op/sec │  ± 4.58 % │ + 2276.97 %             ║
-╚═════════════════════════════════════╧═════════╧══════════════════╧═══════════╧═════════════════════════╝
-
-╔═════════════════════════════════════╤═════════╤══════════════════╤═══════════╤═════════════════════════╗
-║ Slower tests                        │ Samples │           Result │ Tolerance │ Difference with slowest ║
-╟─────────────────────────────────────┼─────────┼──────────────────┼───────────┼─────────────────────────╢
-║ EdDSA - fast-jwt (async)            │    1000 │   6370.58 op/sec │  ± 0.59 % │                         ║
-║ EdDSA - jose (sync)                 │    1000 │   6538.90 op/sec │  ± 0.83 % │ + 2.64 %                ║
-║ EdDSA - fast-jwt (sync)             │    1000 │   7078.93 op/sec │  ± 0.87 % │ + 11.12 %               ║
-║ EdDSA - fast-jwt (async with cache) │   10000 │ 177457.09 op/sec │  ± 5.36 % │ + 2685.57 %             ║
-╟─────────────────────────────────────┼─────────┼──────────────────┼───────────┼─────────────────────────╢
-║ Fastest test                        │ Samples │           Result │ Tolerance │ Difference with slowest ║
-╟─────────────────────────────────────┼─────────┼──────────────────┼───────────┼─────────────────────────╢
-║ EdDSA - fast-jwt (sync with cache)  │   10000 │ 202628.41 op/sec │  ± 3.12 % │ + 3080.69 %             ║
-╚═════════════════════════════════════╧═════════╧══════════════════╧═══════════╧═════════════════════════╝
-```
+See [BENCHMARKS.md](./benchmarks/BENCHMARKS.md)
```

Comment:
why don't we name the file README so when you navigate to the folder it shows immediately?